src/Broadway/EventStore/IlluminateEventStore.php

```diff
@@ -0,0 +1,119 @@
+<?php namespace App\Broadway\EventStore;
+
+use Broadway\Domain\DomainEventStream;
+use Broadway\Domain\DomainEventStreamInterface;
+use Broadway\Domain\DomainMessage;
+use Broadway\Domain\DateTime;
+use Broadway\EventStore\EventStoreInterface;
+use Broadway\EventStore\EventStreamNotFoundException;
+use Broadway\Serializer\SerializerInterface;
+use Illuminate\Config\Repository;
+use Illuminate\Database\DatabaseManager;
+
+/**
+ * Class IlluminateEventStore
+ *
+ * Create a broadway.php config file in your config directory to change the table name
+ *
+ * @author  Dennis Schepers
+ * @package App\Broadway\EventStore
+ */
+class IlluminateEventStore implements EventStoreInterface {
+
+    /**
+     * @var DatabaseManager
+     */
+    protected $databaseManager;
+
+    /**
+     * @var \Broadway\Serializer\SerializerInterface
+     */
+    protected $serializer;
+
+    /**
+     * @var \Illuminate\Contracts\Config\Repository
+     */
+    protected $config;
+
+    /**
+     * Construct the dependancies
+     *
+     * @param \Illuminate\Database\DatabaseManager $databaseManager
+     * @param \Illuminate\Config\Repository $configRepository
+     * @param SerializerInterface $serializer
+     */
+    public function __construct(
+        DatabaseManager $databaseManager,
+        Repository $configRepository,
+        SerializerInterface $serializer
+    ) {
+        $this->databaseManager = $databaseManager;
+        $this->config = $configRepository;
+        $this->table = $this->config->get('broadway.table','event');
```

Comment:
While I've seen the Laravel broadway implementation, I acctualy prefered to implement broadway plain-and-straight. My main project uses Lumen, and the package doesn't mention anything about it.

Concerning the injections:
* While it would be possible to directly inject the Connection, it would require the developer to configure an additional binding within it's application. The current solution would work out of the box.
* The table name can be set with laravels config object, (but it's not a requirement as it defaults to 'event').

The choices I made were basicaly done with the mind set of doing minimal additional configuration when using this implementation.